pkg/cmd/pulumi/about_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Nit: There's still an extra line in here:

```suggestion
// Copyright 2016-2021, Pulumi Corporation.
//
// Licensed under the Apache License, Version 2.0 (the "License");
```